src/components/InvoicesList/InvoicesList.test.js

```diff
@@ -313,4 +313,90 @@ describe('InvoiceList component', () => {
       expect(container.querySelector('.dp-pagination')).not.toBeInTheDocument();
     });
   });
+
+  it('should show an "invoice_text" when DocumentType equal "FC"', async () => {
+    // Arrange
+    const invoicesCollection = {
+      items: [
+        {
+          documentType: 'FC',
+          documentNumber: 'A-0001-00000001',
+          creationDate: new Date('2020-09-25T00:00:00'),
+          dueDate: new Date('2020-09-25T00:00:00'),
+          currency: 'ARS',
+          amount: 1500,
+          filename: 'invoice_2020-10-05_10.pdf',
+          downloadInvoiceUrl: '',
+        },
+      ],
+      totalItems: 1,
+    };
+
+    const dopplerBillingApiClientDouble = {
+      getInvoices: async () => {
+        return { success: true, value: invoicesCollection };
+      },
+    };
+
+    // Act
+    const { getByText } = render(
+      <AppServicesProvider
+        forcedServices={{
+          dopplerBillingApiClient: dopplerBillingApiClientDouble,
+        }}
+      >
+        <IntlProvider>
+          <BrowserRouter>
+            <InvoicesList />
+          </BrowserRouter>
+        </IntlProvider>
+      </AppServicesProvider>,
+    );
+
+    // Assert
+    await waitFor(() => expect(getByText('invoices_list.invoice_text')).toBeInTheDocument());
+  });
+
+  it('should show a "credit_note_text" when DocumentType equal "NC"', async () => {
```

Comment:
Where do you use specifically NC? it's when it's not FC right?

Reply:
Exactly, the DocumentType can be FC or NC.

Reply:
Then perhaps it should be on the line of something like this stick to the code as it is. Since this is a test for the invoice list component. Take this as a mild suggestion. Perhaps I'm looking to much into this 😅 
```suggestion
  it('should show a "credit_note_text" when DocumentType is not "NF"', async () => {
```

Reply:
good, I will change it.

Reply:
Done